src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
Do not create a new Instance of `Random` every time, make it a class-level variable